packages/react/fast-components-react-base/src/data-grid/data-grid-cell.props.ts

```diff
@@ -0,0 +1,30 @@
+import React from "react";
+import {
+    DataGridCellClassNameContract,
+    ManagedClasses,
+} from "@microsoft/fast-components-class-name-contracts-base";
+import { DataGridColumnDefinition } from "./data-grid.props";
+
+export const CellIdKey: string = "data-cellid";
+
+export type DataGridCellManagedClasses = ManagedClasses<DataGridCellClassNameContract>;
+export type DataGridCellUnhandledProps = React.AllHTMLAttributes<HTMLElement>;
+
+export interface DataGridCellHandledProps extends DataGridCellManagedClasses {
+    /**
+     *  Data for this row
+     */
+    rowData: object;
```

Comment:
`object` isn't a very specific data structure - is there a reason this is so vague?

Reply:
This is the actual user provided data object for each row, it could be almost anything .  The column definition specifies the name of field in the object to display in each cell.  So rowData could be {customerId: 1234, name: John Doe, age: 34, etc...} and the column definition would specify which items to display in each cell in the row.

Reply:
Maybe this should be a generic then? This seems really permissive: `T extends {[key: string]: any}`

Reply:
Good suggestion.  I'll see about doing something like that.

Reply:
Unfortunately was not able to figure out a good way to contstrain the type beyond object.